### ISSUE TITLE
Filtering on UTM sources with URL encoded chars does not work

### ISSUE THREAD

**Issue by adriaanvanrossum** [simpleanalytics/roadmap#451]
This URL for example: https://simpleanalytics.com/simpleanalytics.com?start=2020-09-20&end=2020-10-19&utmSources=no%2520code%2520founders

<kbd><img width="545" alt="Screenshot" src="https://user-images.githubusercontent.com/1079135/96481161-762d9000-123b-11eb-8413-fb891fdc6260.png"></kbd>

Do note that we now cleanup all UTM params in https://github.com/simpleanalytics/queue/blob/024daaafae66465b5d48ea4bce899d96a39df0e0/lib/sanitize.js#L74-L78

This cleanup looks like this:

```js
module.exports.cleanParam = (string) => {
  if (typeof string !== "string" || string.length === 0) return string;

  try {
    return decodeURIComponent(
      decodeURIComponent(decodeURIComponent(string))
    ).trim();
  } catch ({ message }) {
    console.error(`[ERROR] cleanParam ${message}: "${string}"`);
    return string;
  }
};
```

So basically it runs `decodeURIComponent` 3 times and trims the param.

**Comment by hellodeibu** [comment]
A fix for this is pending merge and deploy!

**Comment by adriaanvanrossum** [comment]
Works great!

<kbd><img width="1064" alt="Screenshot" src="https://user-images.githubusercontent.com/1079135/98389095-e584eb80-2053-11eb-9e33-077679063dbc.png"></kbd>

https://simpleanalytics.com/simpleanalytics.com?start=2020-10-07&end=2020-11-06&utmSources=no%2520code%2520founders

**Comment by adriaanvanrossum** [comment]
URL looks a bit weird (`no%2520code%2520founders`) but it works. And this works as well: https://simpleanalytics.com/simpleanalytics.com?start=2020-10-07&end=2020-11-06&utmSources=no%20code%20founders
